### PR TITLE
Home Layout

### PR DESCRIPTION
**Pull request by vig1312**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
-import './App.css';
+// Libraries
+    import { Routes , Route } from "react-router-dom";
```
Comment (zgev94) [inline src/App.js]: use single quotes instead of double

Comment (zgev94) [reply src/App.js]: import { Routes , Route } from 'react-router-dom';

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,28 @@
-import './App.css';
+// Libraries
```
Comment (zgev94) [inline src/App.js]: no need in comments here

File: src/Components/HomeLayout/navigation.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// Libraries
+    import { NavLink } from "react-router-dom";
+
+// Data
+    import { navData } from "../../constants/constants";
```
Comment (zgev94) [inline src/Components/HomeLayout/navigation.js]: contstants/constants ? keep correct paths.

File: src/Components/HomeLayout/navigation.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+function Navigation() {
+
+    return (
+        <nav>
+            <ul>
+                {navData.map((item, index) => (
+
+                    <NavLink
+                        className={"nav-links"}
```
Comment (zgev94) [inline src/Components/HomeLayout/navigation.js]: no need in curly braces here  { }, just className="nav-links"

File: src/Components/HomeLayout/navigation.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                        key={index}
+                        to={item.path}
```
Comment (zgev94) [inline src/Components/HomeLayout/navigation.js]: you can destruct item directly in arguments

File: src/Components/topFilmsLayout/filmLists.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
+// Libs
+    import React from 'react';
+    import { Link } from "react-router-dom";
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: don't forget to change imports to single quotes in all files

File: src/Components/topFilmsLayout/filmLists.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+// Datas
+    import {topFilms} from "../../constants/constants";
+
+
+function FilmLists() {
+
+    return (
+        <section>
+
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: no need in empty line

File: src/Components/topFilmsLayout/filmLists.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
+// Libs
+    import React from 'react';
+    import { Link } from "react-router-dom";
+
+// Datas
+    import {topFilms} from "../../constants/constants";
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: empty space before and after while destructing 


import { topFilms }

File: src/Components/topFilmsLayout/filmLists.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+            <h1 className={"top-heading"}>Top Films</h1>
+            <div className={"top-films-container"}>
+
+                {topFilms.map((film, index) => (
+
+                    <Link
+                        key={index}
+                        className={"films-boxes"}
+                        to={`/films/${film.id}`}
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: destruct

File: src/Pages/Home/home.js [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+// Components
+    import Footer from "../../Components/HomeLayout/footer";
+    import Navigation from "../../Components/HomeLayout/navigation";
+
+
+function Home() {
+
+    return (
+
```
Comment (zgev94) [inline src/Pages/Home/home.js]: no need in empty line

File: src/Pages/Home/home.js [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    import { Outlet } from "react-router-dom";
+
+// Components
+    import Footer from "../../Components/HomeLayout/footer";
+    import Navigation from "../../Components/HomeLayout/navigation";
+
+
+function Home() {
+
+    return (
```
Comment (zgev94) [inline src/Pages/Home/home.js]: single line return

File: src/Components/topFilmsLayout/filmLists.js [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+function FilmLists() {
+
+    return (
+        <section>
+
+            <h1 className={"top-heading"}>Top Films</h1>
+            <div className={"top-films-container"}>
+
+                {topFilms.map((film, index) => (
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: {topFilms.map((({ id, title, poster ), index) => (

File: src/App.js [hunk 13]
```diff
@@ -1,10 +1,23 @@
+import { Routes , Route } from 'react-router-dom';
 import './App.css';
```
Comment (zgev94) [inline src/App.js]: keep import order

File: src/Components/HomeLayout/footer.js [hunk 14]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+function Footer() {
+
+    return (
+
```
Comment (zgev94) [inline src/Components/HomeLayout/footer.js]: remove empty lines

File: src/Components/HomeLayout/footer.js [hunk 15]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+function Footer() {
+
+    return (
```
Comment (zgev94) [inline src/Components/HomeLayout/footer.js]: single line return

File: src/Components/HomeLayout/navigation.js [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+import { navData } from '../../Data/constants';
+
+
+function Navigation() {
+
+    return (
+        <nav>
+            <ul>
+                {navData.map(({path,title}, index) => (
+
```
Comment (zgev94) [inline src/Components/HomeLayout/navigation.js]: no need in empty line

File: src/Components/HomeLayout/navigation.js [hunk 17]
```diff
@@ -0,0 +1,27 @@
+import { NavLink } from 'react-router-dom';
+import { navData } from '../../Data/constants';
```
Comment (zgev94) [inline src/Components/HomeLayout/navigation.js]: empty line before this import

File: src/Components/topFilmsLayout/filmLists.js [hunk 18]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { topFilms } from '../../Data/constants';
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js:4]: empty line before this import

File: src/Components/HomeLayout/footer.js [hunk 19]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const Footer = () => (
+
```
Comment (zgev94) [inline src/Components/HomeLayout/footer.js]: remove empty line

File: src/Components/HomeLayout/footer.js [hunk 20]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const Footer = () => (
+    
+    <footer>
+        <h1>This Is Footer</h1>
+    </footer>
+
```
Comment (zgev94) [inline src/Components/HomeLayout/footer.js]: remove empty line

File: src/Components/HomeLayout/navigationTool.js [hunk 21]
```diff
@@ -0,0 +1,25 @@
+import { NavLink } from 'react-router-dom';
+
+import { navData } from '../../Data/constants';
+
+
+function Navigation() {
+
```
Comment (zgev94) [inline src/Components/HomeLayout/navigationTool.js]: remove empty line

File: src/Components/topFilmsLayout/filmLists.js [hunk 22]
```diff
@@ -0,0 +1,34 @@
...
+import { Link } from 'react-router-dom';
+
+import { topFilms } from '../../Data/constants';
+
+
+function FilmLists() {
+
+    return (
+        <section>
+            <h1 className={"top-heading"}>Top Films</h1>
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: remove curly braces

File: src/Components/topFilmsLayout/filmLists.js [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+            <div className={"top-films-container"}>
+
+                {topFilms.map(({id, poster, title}, index) => (
+
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: remove empty line

File: src/Components/topFilmsLayout/filmLists.js [hunk 24]
```diff
@@ -0,0 +1,34 @@
...
+                        className="films-boxes"
+                    >
+                        <img src={poster} width={200} height={300}/>
+                        {title}
+                    </Link>
+
+                    )
+                  )
+                }
+
```
Comment (zgev94) [inline src/Components/topFilmsLayout/filmLists.js]: remove empty line

File: src/Pages/Home/home.js [hunk 25]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+
+import Footer from '../../Components/HomeLayout/footer';
+import Navigation from '../../Components/HomeLayout/navigationTool';
+
+
+const Home = () => (
+
```
Comment (zgev94) [inline src/Pages/Home/home.js]: remove empty line

File: src/App.js (hunk 13 above)
Comment (vig1312) [reply src/App.js]: should css files import  be separate ?

**Review by zgev94 — APPROVED** [review]

**Pull request merged**